packages/types/src/base.ts

```diff
@@ -105,65 +101,9 @@ export interface MinMaxOutput {
   maximum: string
 }
 
-export type QuoteResponse = {
-  price: string
-  to: string
-  data?: string
-  value?: string
-  gas?: string
-  estimatedGas?: string
-  gasPrice?: string
-  protocolFee?: string
-  minimumProtocolFee?: string
-  buyTokenAddress?: string
-  sellTokenAddress?: string
-  buyAmount?: string
-  sellAmount?: string
-  allowanceTarget?: string
-  sources?: Array<SwapSource>
-}
-
-export type Quote<C extends ChainTypes> = {
-  success: boolean
-  statusReason?: string
-  sellAssetAccountId?: string
-  buyAssetAccountId?: string
-  sellAsset: Asset
-  buyAsset: Asset
-  feeData?: QuoteFeeData<C>
-  rate?: string
-  depositAddress?: string // this is dex contract address for eth swaps
-  receiveAddress?: string
-  buyAmount?: string
-  sellAmount?: string
-  minimum?: string | null
-  maximum?: string | null
-  txData?: string
-  value?: string
-  allowanceContract?: string
-  sources?: Array<SwapSource>
-}
-
-export type GetQuoteInput = {
```

Comment:
This still exists in web but is actually defined there: https://github.com/shapeshift/web/blob/0566ee10349ad0925bcb60e52be9292235bcaff1/src/components/Trade/hooks/useSwapper/useSwapper.ts#L34-L37 so this is fine

Reply:
ya I dont think its related to this PR. I also checked the same thing and get the errors but none related to this pr I as far as I can tell